php/example_code/firehose/PutRecordtoDeliveryStream.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ *  ABOUT THIS PHP SAMPLE: This sample is part of the
+ *
+ */
+
+require 'vendor/autoload.php';
+
+use Aws\Firehose\FirehoseClient;
+use Aws\Exception\AwsException;
+
+/**
+ * Creating an Amazon Kinesis Firehose client.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+//Create a KinesisClient
+$FirehoseClient = new Aws\Firehose\FirehoseClient([
+    'profile' => 'default',
+    'version' => '2015-08-04',
+    'region'  => 'us-east-2'
+]);
+
+$name = "my_stream_name";
+$content = "\"ticker_symbol\":\"QXZ\", \"sector\":\"HEALTHCARE\", \"change\":-0.05, \"price\":84.51}";
```

Comment:
Using single quotes (`'`) means you don't have to escape the double quotes inside.